app/static/js/base.js

```diff
@@ -18,3 +18,23 @@ $(document).ready(() => {
 		$(this).closest('div').remove();
 	});
 });
+
+function openTab(evt, level) {
+  var i, tabcontent, tablevel;
+
+  // Get all elements with class="tabcontent" and hide them
+  tabcontent = document.getElementsByClassName("tabcontent");
+  for (i = 0; i < tabcontent.length; i++) {
+    tabcontent[i].style.display = "none";
+  }
```

Comment:
Isn't this done by [here](https://github.com/KuriusMTL/HackItForward/pull/59/files#diff-6f9c621d622dc2bc991e46a81c09a46797cd4e29acbd536531926d649a4c540aR259)?

Reply:
no, it's different. The code won't work if you remove this Javascript code.